dex-it/src/test/scala/com/wavesplatform/it/sync/networking/NetworkIssuesTestSuite.scala

```diff
@@ -40,6 +43,41 @@ class NetworkIssuesTestSuite extends WsSuiteBase with HasToxiProxy {
     clearOrderBook()
   }
 
+  "DEXClient should place orders despite of short time disconnect from network" in {
+
+    val orders = (0 to 100).map { i =>
+      mkOrder(alice, wavesUsdPair, OrderType.SELL, 1.waves, 100 + i)
+    }
+
+    Await.ready(
```

Comment:
If something fails in for comprehension, we won't see this, because `Await.ready` wait for `Future` to be ready. It won't throw an exception if it failed. There are rare cases for `Await.ready`, use `Await.result` always instead.

Reply:
ok